source/developers/cook-books/how-tos/web-app-manifest-use-credentials.rst

```diff
@@ -0,0 +1,57 @@
+:is-up-to-date: True
+
+.. index:: Enable Use of Cookies and Credentials by the Web App Manifest, Web App Manifest, Manifest
+
+===================================================
+Enabling Use of Credentials by the Web App Manifest
+===================================================
+
+Some sites uses a web app manifest, a file that describes the site.
+
+From `Google <https://developers.google.com/web/fundamentals/web-app-manifest>`_:
+
+   The web app manifest is a simple JSON file that tells the browser about your web application and how it should behave when ‘installed’ on the user’s mobile device or desktop.
+
+To tell the browser about the manifest, a link tag is added to all pages of your site:
+
+.. code-block:: html
+
+   <link rel="manifest" href="/manifest.json">
```

Comment:
These code blocks blend too much into the content (in the final render of this docs page). Would be nice to add what ever makes the code block wrapped in a box with a border a such.